packages/tools/src/tools/mcp.ts

```diff
@@ -60,10 +78,27 @@ class MCPClient {
     this.verbose = options.verbose ?? false;
     this.toolNamePrefix = options.toolNamePrefix;
     if ("url" in options) {
-      this.transport = new SSEClientTransport(
-        new URL(options.url),
-        options as SSEClientTransportOptions,
-      );
+      // If useSEE is not provided, default to true
+      // to avoid breaking changes
+      if (options.useSSE === undefined) {
+        options.useSSE = true;
+      }
+      if (options.useSSE) {
+        // Show deprecation warning
+        console.warn(
+          "SSE transport will be soon deprecated. " +
+            "Please use StreamableHTTPClientTransport instead.",
+        );
+        this.transport = new SSEClientTransport(
+          new URL(options.url),
+          options as SSEClientTransportOptions,
+        );
```

Comment:
if we useSSE this will always use the old `SSEClientTransport` code - why not removing it?